src/Dfc.CourseDirectory.WebV2/Features/BulkUpload/CoursesPublishFile.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.BulkUpload.CoursesPublishFile.ViewModel
+@{ ViewBag.Title = "Publish your file"; }
+<div class="govuk-grid-row" data-sf-element="Row">
+    <div class="govuk-grid-column-two-thirds">
+        <form class="govuk-!-margin-bottom-9" id="DownloadErrorFileForm" asp-controller="BulkuploadApprenticeships" asp-action="PublishYourFile" method="post">
```

Comment:
Nit: is the ID attribute required?

Reply:
This is copied from the original view, I'm inclined to make minimal changes to the cshtml in this PR while we switch to v2, and leave more thorough cleanup for another day. You might also roll an eye at apprenticeship reference in there, which is ignored because the button has a route too. Nrrghh.

Reply:
Okay, I agree actually. Didn't find any references. Killed it.